app/src/main/java/net/gsantner/markor/util/MarkorContextUtils.java

```diff
@@ -46,13 +47,27 @@ public <T extends GsContextUtils> T applySpecialLaunchersVisibility(final Contex
         return thisp();
     }
 
+    private static int getIconResForFile(final File file) {
```

Comment:
suggestion: add @ nullable or nonnull, if it's nullable then handle if (null) in first condition

Reply:
Fixed